### PR TITLE
Estimate runtime refactoring

### PR DESCRIPTION
**Pull request by yippieyaray**
This is my refactoring of the estimate_runtime function to fix "Runtime estimation problem #115 (https://github.com/TomWhitwell/SlowMovie/issues/115)"
It's not my quick fix from yesterday. I remove all unnecessary loops and recursions as well.

### CODE REVIEW

**Comment by robweber** [comment]
The changes make sense to me. I ran the new function in it's own file with a variety of different increments and frame values without issue. The only spot I can find this function is called is when the [remaining time is logged](https://github.com/TomWhitwell/SlowMovie/blob/main/slowmovie.py#L378). For it's intended purpose I think making it as lightweight as possible is a good change. 

@qubist - I think you may have originally added this function. Any thoughts?

**Comment by qubist** [comment]
If merged, this closes #115

File: slowmovie.py [hunk 1]
```diff
@@ -165,35 +165,27 @@ def get_random_video(viddir):
 
 
 # Calculate how long it'll take to play a video.
-# output value: "d[ay]", "h[our]", "m[inute]", "s[econd]", "all"; omit for an automatic guess
-def estimate_runtime(delay, increment, frames, output="guess"):
-    # Recurse to generate all estimates in one string
-    if output == "all":
-        return f"{estimate_runtime(delay, increment, frames, 's')} / {estimate_runtime(delay, increment, frames, 'm')} / {estimate_runtime(delay, increment, frames, 'h')} / {estimate_runtime(delay, increment, frames, 'd')}"
-
+# output value: "d[ay]", "h[our]", "m[inute]", "s[econd]", false; omit for an automatic guess
```
Comment (qubist) [inline slowmovie.py:168]:
```suggestion

```

**Review by qubist — CHANGES_REQUESTED** [review]
This is a very elegant refactor of the function! Thank you for your contribution, @yippieyaray. I'm ready to approve changes after the comment line I mentioned is removed.

**Comment by qubist** [comment]
Thanks for moving forward with this, @missionfloyd !

**Review by qubist — APPROVED** [review]
I'm ready for this to be merged

**Pull request merged**